src/app/templates/client/Client.jsx

```diff
@@ -94,6 +127,7 @@ function Client() {
       <Dialogs />
       <EmojiBoardOpener />
       <ReusableContextMenu />
+      <DragDrop isOpen={dragCounter !== 0} />
```

Comment:
Currently DragDrop window appears event when there is a modal window(like settings) is open. this can confuse people as they may think that they are dropping it in modal window

Reply:
`navigation.isRawModalVisible` can be used to know if there is any modal window opened

Reply:
I tried using it but adding a log statement to the setter function shows that it's really flaky if you want to call it that. Close and reopen the settings menu and the value is false. Things like that. I'll look what I can do

Reply:
which setter function?

Reply:
`navigation.setIsRawModalVisible`
I only added the log statement locally for testing btw in case you're trying to find it in the commits.